x-pack/plugins/spaces/public/lib/spaces_manager.ts

```diff
@@ -54,6 +56,10 @@ export class SpacesManager extends EventEmitter {
       .catch(() => this._displayError());
   }
 
+  public redirectToSpaceSelector() {
+    window.location.href = this.rootBasePath;
```

Comment:
`this.rootBasePath` can be `""` and in that situation, this isn't working, we wanna be using `"/"` instead.